11-Backbonejs/07-JavaScript-Objects-to-Backbone-Models.md

```diff
@@ -0,0 +1,230 @@
+# Scrabble-JS Total Conversion to Backbone.js!
+
+We've already done Scrabble & Player objects in Javascript.  Now we'll take those generic objects and turn'em into Backbone Models! 
+
+## Learning Goals
+With this live code/lecture you should develop the following skills:
+- Learn how Backbone Models compare with pure Javascript Objects.
+- Learn how to adjust Jasmine tests for Backbone models
+
+
+### Baseline Setup
+The starting code with Scrabble & Player objects are located [here](https://github.com/CheezItMan/scrabble-js-to-backbone).  The file has two JavaScript files `src/scrabble.js` and `src/player.js`.  It also has two spec files `spec/scrabble_spec.js` and `spec/player_spec.js`.  
+
+Go ahead and fork & clone the repo.  
+
+Then take a minute to look at the `scrabble.js` and `player.js` together with your partner to get a bit familiar with them.  
+
+#### Commands
+Because this uses the scaffold you still need to run `npm install` after forking and cloning the repository. **You do not need to run** `npm start` as we won't have a web application.
+
+To run your tests you should use the command `npm test`.
```

Comment:
We should probably encourage them to run the tests once at the beginning to confirm they're all working. Then later on after we convert each object and update the tests accordingly, we can  have them run the tests again to confirm they're all still working. 

I don't know that they've had much exposure to actually using tests to support a major refactor (beyond us saying it), so that could be a good eye-opener for them.